config/locales/geoblacklight.en.yml

```diff
@@ -15,7 +15,7 @@ en:
       error_with_url: 'Sorry, the requested file could not be downloaded. Try downloading it directly from: %{link}'
       export_formats: 'Export Formats'
       export_link: 'Export'
-      export_shapefile_link: 'Shapefile'
+      export_shapefile_link: 'WGS 84 Shapefile'
```

Comment:
Can we guarantee that this is a WGS 84 shapefile for all metadata?

Reply:
Currently, that's how data is reprojected, when it's fetched from GeoServer.  I'm not sure if that can be changed, though.  cc @tbrittnacher